mangaki/mangaki/utils/anidb.py

```diff
@@ -45,98 +40,43 @@ def search(self, q):
     results = []
     animetitles = BeautifulSoup(r.text, 'xml').animetitles
     for anime in animetitles.find_all('anime'):
-      results.append(Anime({
-        'id': int(anime['aid']),
-        'title': str(anime.find('title', attrs={'type': "official"}).string),
-        'picture': "http://img7.anidb.net/pics/anime/" + str(anime.find('picture').string),
-      }, partial=True, updater=lambda: self.get(anime.id)))
-
+      results.append(
+        (int(anime['aid']),
+        str(anime.find('title', attrs={'type': "official"}).string))
+      )
     return results
 
-  def get(self, id):
+  def get(self, anidb_aid):
     """
     Allows retrieval of non-file or episode related information for a specific anime by AID (AniDB anime id).
     http://wiki.anidb.net/w/HTTP_API_Definition#Anime
     """
-    id = int(id) # why?
+    anidb_aid = int(anidb_aid) # why?
```

Comment:
Ensure that the `anidb_aid` given is a number / integer. Must have been put to interface with strange code which would call this function using string, it's not necessary, we could just assume that `anidb_aid` is of type int using type hinting.